src/static/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+	"short_name": "OpenCollective",
+	"name": "OpenCollective",
```

Comment:
We usually spell "OpenCollective" with a space "Open Collective".

If space is not available, it's more like an identifier and is usually lowercased: "opencollective"

Reply:
ok, corrected